compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -612,10 +705,31 @@ struct QuantizeWeights final : public luci::CircleNodeMutableVisitor<bool>
   }
 };
 
+void quant_instnorm(luci::CircleInstanceNorm *node, loco::DataType output_type,
+                    QuantizationGranularity granularity)
+{
+  auto gamma = loco::must_cast<luci::CircleConst *>(node->gamma());
+  auto beta = loco::must_cast<luci::CircleConst *>(node->beta());
+  assert(gamma->dtype() == loco::DataType::FLOAT32);
+  assert(beta->dtype() == loco::DataType::FLOAT32);
+
+  if (granularity == QuantizationGranularity::LayerWise)
+  {
+    quant_const(gamma, output_type);
+    quant_const(beta, output_type);
+  }
+  else
+  {
+    quant_const_per_channel(gamma, output_type);
+    quant_const_per_channel(beta, output_type);
+  }
```

Comment:
```suggestion
  else if (granularity == QuantizationGranularity::ChannelWise)
  {
    quant_const_per_channel(gamma, output_type);
    quant_const_per_channel(beta, output_type);
  }
  else
    throw std::runtime_error("Quantization granularity must be either 'layer' or 'channel'");
```

Reply:
if `QuantizationGranularity` may provide another type then this change would be good

Reply:
I've updated the code.  Thanks.